packages/astro-og-canvas/tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "declaration": true,
     "noEmit": false,
+    "allowImportingTsExtensions": false,
```

Comment:
What was the reason for adding this?

Reply:
Without it `prepublish` fails:

```bash
$ npm run prepublish

> @root/astro-og-canvas@0.0.1 prepublish
> npm run build --workspace=astro-og-canvas


> astro-og-canvas@0.1.8 build
> tsc

error TS5096: Option 'allowImportingTsExtensions' can only be used when either 'noEmit' or 'emitDeclarationOnly' is set.
```

Reply:
Thanks for the clarification! I guess that was caused by a change to the Astro tsconfig we extend.